src/styles/s-forms.css

```diff
@@ -315,6 +315,20 @@ input:disabled {
   pointer-events: auto;
 }
 
+.is-warning {
```

Comment:
Not sure why you need !important, so maybe the following will help :) :) :):
https://chat.openai.com/share/7719aa77-cfce-44ed-b0c2-333be686a05e

we can discuss it of course :).

Reply:
In this case, it does not :'(  https://github.com/kbss-cvut/s-forms/pull/207#issuecomment-1948884551

Reply:
Maybe we can discuss if you are out of ideas.